packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -31,6 +33,15 @@ const selectors = {
 };
 const EXTENDED_TIMEOUT = 90 * 1000;
 
+export type OpenInlineInserterDelegate = ( editor: Locator ) => Promise< void >;
+interface BlockInserter {
```

Comment:
I don't think we need to export this and apply it to the components. Since TypeScript supports duck typing (of sorts), if the signatures of those component classes changes so that we can't consolidate them here with this interface, we'll get typing errors!